app/src/main/java/com/dimelo/sampleapp/RcConfig.java

```diff
@@ -18,7 +22,12 @@ public static Dimelo setupDimelo(Context context) {
         dimelo.initWithApiSecret(secret, domainName, null);
         dimelo.setDebug(true);
         dimelo.setUserName("John Doe");
-        dimelo.setThreadsEnabled(true);
+        boolean isThreadEnabled = RcConfig.getBooleanValueFromSharedPreference(context,RC_THREAD_ENABLED);
```

Comment:
Isn't it possible to run the linter in the sample app too?

```suggestion
        boolean isThreadEnabled = RcConfig.getBooleanValueFromSharedPreference(context, RC_THREAD_ENABLED);
```